lib/src/signed_video_tlv.c

```diff
@@ -256,28 +277,25 @@ decode_general(signed_video_t *self, const uint8_t *data, size_t data_size)
     }
     bytes_to_version_str(self->code_version, self->authenticity->version_on_signing_side);
 
+    if (version == 2) {
+      // Read bool flags
+      uint8_t flags = 0;
+      data_ptr += read_8bits(data_ptr, &flags);
+      gop_info->has_timestamp = flags & 0x01;
+      self->latest_validation->has_timestamp = gop_info->has_timestamp;
```

Comment:
There is actually no point in storing the intermediate values in `gop_info`. It is not used elsewhere. You can store directly into `latest_validation`

Reply:
No